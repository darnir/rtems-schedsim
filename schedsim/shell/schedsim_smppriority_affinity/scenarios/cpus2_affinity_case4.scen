#
#  
#
rtems_init 2
task_create smp1 3 -a 2 
task_create smp2 2 -a 1

# GOAL: want 0x0a010002 - 0x0a010001 as heir and executings
# GOAL:   on CPUs 0 -1 respectively
cpus
