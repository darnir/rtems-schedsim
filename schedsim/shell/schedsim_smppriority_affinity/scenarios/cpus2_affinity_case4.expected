Script File               : scenarios/cpus2_affinity_case4.scen
verbose                   : 1
==> 1: #
==> 2: #  
==> 3: #
==> 4: rtems_init 2
  CPU 0: Thread Heir 0x09010001 priority=255
  CPU 0: Thread Executing 0x09010001 priority=255
  CPU 1: Thread Heir 0x09010002 priority=255
  CPU 1: Thread Executing 0x09010002 priority=255
==> 5: task_create smp1 3 -a 2 
Creating task with:  Preemption: yes Timeslicing: no
Task (smp1) created: id=0x0a010001, priority=3
Task (0x0a010001) Set affinity=0x00000002
Task (smp1) starting: id=0x0a010001, priority=3
  CPU 1: Thread Heir 0x0a010001 priority=3
  CPU 1: Thread Executing 0x0a010001 priority=3
==> 6: task_create smp2 2 -a 1
Creating task with:  Preemption: yes Timeslicing: no
Task (smp2) created: id=0x0a010002, priority=2
Task (0x0a010002) Set affinity=0x00000001
Task (smp2) starting: id=0x0a010002, priority=2
  CPU 0: Thread Heir 0x0a010002 priority=2
  CPU 0: Thread Executing 0x0a010002 priority=2
==> 7: 
==> 8: # GOAL: want 0x0a010002 - 0x0a010001 as heir and executings
==> 9: # GOAL:   on CPUs 0 -1 respectively
==> 10: cpus
=== CPU Status
          EXECUTING      /   HEIR             / SWITCH NEEDED
  CPU 0: 0x0a010002 @  2 / 0x0a010002 @  2          false
  CPU 1: 0x0a010001 @  3 / 0x0a010001 @  3          false
=== End of Ready Set of Threads
